submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/stats/web/StatsResource.kt

```diff
@@ -42,6 +44,12 @@ class StatsResource(
         @PathVariable accNo: String
     ): SubmissionStat = submissionStatsService.findByAccNoAndType(accNo, SubmissionStatType.valueOf(type.uppercase()))
 
+    @PostMapping
+    @ResponseBody
+    fun register(
+        @RequestBody stat: SingleSubmissionStat
```

Comment:
it looks to me extrange that a class from persistence package is used directly in web layer.